charts/openshift-metering/values.yaml

```diff
@@ -544,6 +544,15 @@ presto:
             useInstanceCredentials: null
 
         extraConnectorFiles: []
+        prometheus:
+          enabled: true
+          uri: ""
+          chunkSizeDuration: null
+          maxQueryRangeDuration: null
+          cacheDuration: null
```

Comment:
I think we should nest lines 550-552 under `config`, (and update the charts) i.e.:

```yaml
config:
  chunkSizeDuration: null
  maxQueryRangeDuration: null
  cacheDuration: null
```

wdyt?

Reply:
well if we do that it should include `uri` too, as with that one under config: we'd have the full grouping of presto-prom connector configs, it would call them out as separate from `enabled`, which is not.

Reply:
Done in 04273d69